.travis.yml

```diff
@@ -4,33 +4,46 @@ sudo: true
 
 addons:
   firefox: "47.0.1"
-  postgresql: "9.3"
+  postgresql: "9.5"
   apt:
     packages:
-      - oracle-java8-installer
-      - oracle-java8-set-default
 
 cache:
   directories:
     - $HOME/.composer/cache
     - $HOME/.npm
 
-php:
- - 7.0
- - 7.1
- - 7.2
+dist: trusty
 
 env:
- global:
-  - MOODLE_BRANCH=MOODLE_35_STABLE
- matrix:
-  - DB=pgsql
-  - DB=mysqli
+ include:
+  - php: 7.1
+    env: DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE NODEJS=14
+  - php: 7.1
+    env: DB=pgsql  MOODLE_BRANCH=MOODLE_35_STABLE NODEJS=14
+  - php: 7.1
+    env: DB=pgsql  MOODLE_BRANCH=MOODLE_36_STABLE
+  - php: 7.1
+    env: DB=pgsql  MOODLE_BRANCH=MOODLE_37_STABLE NODEJS=14
+  - php: 7.1
+    env: DB=pgsql  MOODLE_BRANCH=MOODLE_38_STABLE NODEJS=14
+  - php: 7.1
+    env: DB=mysqli MOODLE_BRANCH=MOODLE_39_STABLE NODEJS=14
+  - php: 7.2
+    env: DB=mysqli MOODLE_BRANCH=master NODEJS=14
+  - php: 7.2
+    env: DB=pgsql  MOODLE_BRANCH=master NODEJS=14
+
 
 before_install:
   - phpenv config-rm xdebug.ini
-  - nvm install 8.9
-  - nvm use 8.9
+  - if [ "$NODEJS" = 14 ]; then
```

Comment:
I'm trying to remember what node could be used for with CI.
We certainly don't need to do anything with node with Bitbucket pipelines CI.

Reply:
This is used for the js compilation within moodle, this is some boilerplate travis that we add to most of our plugins so that we can support some older non supported branches like 3.6 alongside 3.9 / master

The swap to version 14 has causes us a bit of grief with all our plugins:

https://docs.moodle.org/dev/Javascript_Modules#Install_NVM_and_Node

Reply:
Thanks for the info :-)